File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/wrapper/exception/PacketWriteException.java
package com.gmail.marcosav2010.communicator.packet.wrapper.exception;

import java.io.IOException;

public class PacketWriteException extends IOException {

    private static final long serialVersionUID = 3252190216554456161L;

    public PacketWriteException(Exception exception) {
        super(exception);
    }

    public PacketWriteException(String msg) {
        super(msg);
    }
}

File: core/src/main/java/com/gmail/marcosav2010/communicator/packet/wrapper/exception/PacketWriteException.kt
package com.gmail.marcosav2010.communicator.packet.wrapper.exception

import java.io.IOException

class PacketWriteException : IOException {

    constructor(exception: Exception?) : super(exception)
    constructor(msg: String?) : super(msg)

    companion object {
        private const val serialVersionUID = 3252190216554456161L
    }
}